environment.yml

```diff
@@ -17,6 +17,7 @@ dependencies:
   - requests=2.31.0
   - click=8.1.7
   - quarto=1.5.57
+  - tabulate=0.9.0
```

Comment:
Adding this alone would not trigger the docker github action to trigger unfortunately